### DISCUSSION TITLE
Show Recently Used Files

### DISCUSSION THREAD

**Discussion by qhga** [dominiksta/wournal#2] [Ideas]
It would be great if the menu `Burger Menu > File` would have an additional option `Load Recently Used File > [Hover Menu: Files...]` that shows the last `n` files that were edited within wournal.

**Comment by dominiksta** [comment]
Thanks for the suggestion! This is now implemented and coming in the next release of Wournal. (Implemented in 9dfca7958cd9e142a056015ce8e00850d4a2c972.)
